client/src/components/EventEdit/Header/index.js

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import Moment from 'react-moment';
+import PropTypes from "prop-types";
+import { FiPrinter, FiSettings, FiDownload } from "react-icons/fi";
+import styled from "styled-components";
+
+import IconButton from "../../Common/IconButton";
+import Stack from "../../Common/Stack";
+import {
+  theme1,
+  fontSize64,
+  fontSize24,
+  spacing24,
+} from "../../../resources/style-constants";
+
+const Container = styled.div`
+  background-color: ${props => props.bgcolor || theme1};
+`;
+
+const Spacer = styled.div`
+  height: ${spacing24};
+`;
+
+const TitleContainer = styled.h1`
+  font-size: ${fontSize64};
+  margin: 0;
+`;
+
+const DateContainer = styled.h5`
+  font-size: ${fontSize24};
+  opacity: 0.5;
+  margin: 0;
+`;
+
+const ButtonBarContainer = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+`;
+
+/**
+ * The header which appears at the top of the event-edit page.
+ */
+const Header = ({
+  title,
+  date,
+  dateDisplay,
+  backgroundColor,
+  onPressDownload,
+  onPressPrint,
+  onPressSettings,
+  ...props
+}) => {
+  const effectiveDate = dateDisplay ?
+    (<>dateDisplay</>) :
+    (<Moment format="MM/DD/YYYY">{date}</Moment>);
+
+  return (
+    <Container {...props}>
```

Comment:
I used the stack as a child property and passed the props on into the container. This will allow the caller to provide padding to the colored-background as needed and make use of the reusable stack widget